build.gradle

```diff
@@ -259,11 +261,11 @@ ext.getPluginResource = { download_to_folder, download_from_src ->
 }
 
 String baseName = "asynSearchCluster"
-String bwcVersionShort = "2.11.0"
+String bwcVersionShort = "2.12.0"
 String bwcVersion = bwcVersionShort + ".0"
 String bwcFilePath = "src/test/resources/org/opensearch/search/asynchronous/bwc/"
 String bwcRemoteFile = "https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/"+ bwcVersionShort + "/latest/linux/x64/tar/builds/opensearch/plugins/opensearch-asynchronous-search-"+ bwcVersion +".zip"
-getPluginResource(bwcFilePath + bwcVersion, bwcRemoteFile)
+//getPluginResource(bwcFilePath + bwcVersion, bwcRemoteFile)
```

Comment:
why are we commenting this

Reply:
Because we are providing it offline. It was unable to download the zip from remote. So inorder to fix build we downloaded it from repository and put in test/resource folder.